tests/integration/ceph_mgr_test.go

```diff
@@ -50,9 +49,6 @@ func TestCephMgrSuite(t *testing.T) {
 		t.Skip()
```

Comment:
The test is still being skipped, likely because of this check for the official build. Since we are using github actions now, this check is not needed anymore.